src/esp/geo/CoordinateFrame.h

```diff
@@ -40,6 +40,10 @@ class CoordinateFrame {
   //! to coordinates in this CoordinateFrame
   quatf rotationWorldToFrame() const;
 
+  //! Returns quaternion representing rotation taking world coordinates
+  //! to coordinates in this CoordinateFrame
```

Comment:
Perhaps clearer to say `rotation taking direction vectors in world coordinates to direction vectors in this CoordinateFrame` (since applying just this quat to world coordinates without accounting for translations doesn't account for the full `world coordinates to CoordinateFrame coordinates` transformation)